exampleCourse/questions/element/multipleChoice/question.html

```diff
@@ -1,263 +1,260 @@
+
+    <pl-question-panel>
+      <p>Here, you can observe the behavior of the <code>score</code> attribute in <code>pl-answer</code>.
+        The score for a question the percentage of points given to it if selected (default is 0 for
+        incorrect answers and 1 for correct ones). This also allows no submission by the student, done by
+        setting <code>allow-blank="true"</code>.</p>
+      <p> What is the color of the sky? </p>
+    </pl-question-panel>
+
+    <pl-multiple-choice answers-name="sky-color9" allow-blank="true">
+      <pl-answer correct="true" feedback="Correct! The sky is indeed blue.">Blue</pl-answer>
+      <pl-answer>Pink</pl-answer>
+      <pl-answer score=0.5>Purple</pl-answer>
+      <pl-answer feedback="Not quite" score=0.1>Orange</pl-answer>
+      <pl-answer>Yellow</pl-answer>
+      <pl-answer score=0.3>Brown</pl-answer>
+      <pl-answer score=0.2>Red</pl-answer>
+    </pl-multiple-choice>
+  </div>
 </div>
 
 <div class="card my-2">
-    <div class="card-header">
-        Part 9
-    </div>
-
-    <div class="card-body">
-
-        <pl-question-panel>
-            <p>Here, you can observe the behavior of the <code>score</code> attribute in <code>pl-answer</code>.
-            The score for a question the percentage of points given to it if selected (default is 0 for
-            incorrect answers and 1 for correct ones). This also allows no submission by the student, done by
-            setting <code>allow-blank="true"</code>.</p>
-
-            <p> What is the color of the sky? </p>
-        </pl-question-panel>
-
-        <pl-multiple-choice answers-name="sky-color9" allow-blank="true">
-            <pl-answer correct="true" feedback="Correct! The sky is indeed blue.">Blue</pl-answer>
-            <pl-answer>Pink</pl-answer>
-            <pl-answer score=0.5>Purple</pl-answer>
-            <pl-answer feedback="Not quite" score=0.1>Orange</pl-answer>
-            <pl-answer>Yellow</pl-answer>
-            <pl-answer score=0.3>Brown</pl-answer>
-            <pl-answer score=0.2>Red</pl-answer>
-        </pl-multiple-choice>
-
-    </div>
+  <div class="card-header">Part 9</div>
+
+  <div class="card-body">
+    <pl-question-panel>
+      <p>
+        Here, we observe the dropdown display option. This lets students input an answer from a
+        dropdown list instead of choosing from multiple choice radio buttons. Note that the
+        dropdown list supports $\LaTeX$ rendering.
+      </p>
+
+      <p>What is the color of the sky?</p>
+    </pl-question-panel>
+
+    <p>
+      The sky is
+      <pl-multiple-choice answers-name="sky-color10" display="dropdown" allow-blank="true" order="random">
+        <pl-answer correct="true" feedback="Correct! The sky is indeed blue.">Blue ($470nm$)</pl-answer>
+        <pl-answer score="0.5">Purple ($400nm$)</pl-answer>
+        <pl-answer feedback="Hint: The correct answer is a complementary color to this one." score="0.1">Orange
+          ($600nm$)</pl-answer>
```

Comment:
A final nit:

```suggestion
        <pl-answer feedback="Hint: The correct answer is a complementary color to this one." score="0.1">
          Orange ($600nm$)
        </pl-answer>
```

Reply:
It indeed does not affect the formatting, from what I can tell.